### PR TITLE
suppressing grpc server startup logs

### PR DESCRIPTION
**Pull request by brettsam**

### CODE REVIEW

File: src/Azure.Functions.Cli/Azure.Functions.Cli.csproj [hunk 1]
```diff
@@ -154,7 +154,7 @@
     <PackageReference Include="YamlDotNet" Version="6.0.0" />
   </ItemGroup>
   <ItemGroup Condition="'$(NoWorkers)' != 'true'">
-    <PackageReference Include="Microsoft.Azure.Functions.JavaWorker" Version="1.8.2-SNAPSHOT" />
+    <PackageReference Include="Microsoft.Azure.Functions.JavaWorker" Version="1.8.2-test" />
```
Comment (amamounelsayed) [inline src/Azure.Functions.Cli/Azure.Functions.Cli.csproj:157]: Why do we use 1.8.2-test?

Comment (brettsam) [reply src/Azure.Functions.Cli/Azure.Functions.Cli.csproj:157]: well, that was an accident.... i'll revert. (trying to write code while in another meeting :-))

**Review by Francisco-Gamino — APPROVED** [review]
LGTM

**Comment by Francisco-Gamino** [comment]
@michaelpeng36 -- Could you please re-enable the Node test case after this PR is merged? Thank you.

**Pull request merged**